lint/lint.py

```diff
@@ -398,6 +403,15 @@ def process_errors(path, errors):
         if not os.path.exists(abs_path):
             continue
 
+        ignored = False
+        for file_match in ignored_files:
```

Comment:
I think one would usually write this as `any(fnmatch.fnmatch(path, file_match) for file_match in ignored_files)`